examples/prom/mixed_nonlinear_diffusion.cpp

```diff
@@ -794,8 +794,8 @@ int main(int argc, char *argv[])
         vector<int> sample_dofs(nldim);  // Indices of the sampled rows
         CAROM::DEIM(FR_librom,
                     nldim,
-                    &sample_dofs[0],
-                    &num_sample_dofs_per_proc[0],
+                    sample_dofs,
```

Comment:
@dylan-copeland I had to make some changes in this commit to the mixed nonlinear diffusion example to get the master branch to build. Does this look good to you?

Reply:
Yes, thanks for updating.